org/postgresql/jdbc2/AbstractJdbc2ResultSet.java

```diff
@@ -488,7 +488,7 @@ public int getConcurrency() throws SQLException
                 return connection.getTimestampUtils().toDateBin(tz, this_row[col]);
             } else if (oid == Oid.TIMESTAMP || oid == Oid.TIMESTAMPTZ) {
                 // JDBC spec says getDate of Timestamp must be supported
-                return connection.getTimestampUtils().convertToDate(getTimestamp(i, cal), tz);
+                return connection.getTimestampUtils().convertToDate(getTimestamp(i, Calendar.getInstance(TimeZone.getTimeZone("UTC"))), tz);
```

Comment:
I do not like calendar.getinstance.
If "UTC" calendar is required, it should be cached inside `TimestampUtils`.